R/inline_text.R

```diff
@@ -570,3 +570,53 @@ inline_text.tbl_survfit <-
       ) %>%
       pull(.data$statistic)
   }
+
+
+#' Report statistics from cross table inline
+#'
+#' \Sexpr[results=rd, stage=render]{lifecycle::badge("experimental")}
+#' @param x a `tbl_cross` object
+#' @param row_level Level of the row variable to display.
+#' Can also specify the 'Unknown' row. Default is `NULL`
+#' @param col_level Level of the column variable to display.
+#' Can also specify "`p.value`" for the p-value and "`stat_0`" for Total column.
+#' @inheritParams inline_text.tbl_summary
+#'
+#' @return A string reporting results from a gtsummary table
+#' @family tbl_cross tools
+#' @export
+#' @examples
+#' tbl_cross <-
+#'   tbl_cross(trial, row = trt, col = response) %>%
+#'   add_p()
+#'
+#' inline_text(tbl_cross, row_level = "Drug A", col_level = "1")
+#' inline_text(tbl_cross, row_level = "Total", col_level = "1")
+#' inline_text(tbl_cross, col_level = "p.value")
+
+inline_text.tbl_cross <-
```

Comment:
It's great that this works for tbl_cross now 

I'm getting different behavior when I only select a `row_level` vs  when I only select a `col_level`. It might be nice if these were consistent (unless user selects `"p.value"`), or if there were a consistent/ more informative error. 

`tbl <- trial %>% tbl_cross(response, trt)`
`inline_text(tbl, col_level = "Drug A")`
`inline_text(tbl, row_level = 1)`

Maybe if the user only selects `row_level` but not `col_level`, we could return the total and vis versa for col?

The default is not listed on docs for col_level. 

Also, can we allow the user to specify `"Total"` for `col_level` too (along with `"stat_0"`)?

Reply:
You're right this needs updating, but I am not sure what the best defaults should be?

1. being able to specify Total and stat_0 makes sense.
1. perhaps we should require both to row and col to be specified?  I don't want to default to total, since we'll add later the margin option to include/exclude the margins.